installer/terraform/scripts/configureESEndpoint.sh

```diff
@@ -3,7 +3,8 @@
 # TODO This should be in Python + drop curl dep
 
 ES_ENDPOINT=$1
```

Comment:
Can we port this to python since we're touching it and it's already an ES related change?

Reply:
Sure

Reply:
Done